client/components/domains/register-domain-step/index.jsx

```diff
@@ -166,6 +166,12 @@ var RegisterDomainStep = React.createClass( {
 	},
 
 	searchForm: function() {
+		var placeholderText = this.translate( 'Enter a domain or keyword', { textOnly: true } );
```

Comment:
Removed in https://github.com/Automattic/wp-calypso/commit/c0981a2630afc656dfa44778fbc976e4db0066a2.